### PR TITLE
SAR-8701 Update Read me

### PR DESCRIPTION
**Pull request by EviNikolaidou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,13 @@ optServiceName will default to `Entity Validation Service` if the field is not p
 
 All other fields must be provided.
 
+The businessVerificationCheck field allows the calling service to bypass the verification and 
+continue to register where a successful match is found. By default "businessVerificationCheck" will be false.
+
 ```
 {
     "continueUrl" : "/test",
+    "businessVerificationCheck" : false ,
```
Comment (HarrNM) [inline README.md:35]: I think these have been set to true by default, to not break the current Sole Trader flows

File: README.md [hunk 2]
```diff
@@ -45,9 +49,14 @@ optServiceName will default to `Entity Validation Service` if the field is not p
...
 All other fields must be provided.
 
+
+The businessVerificationCheck field allows the calling service to bypass the verification and
+continue to register where a successful match is found. By default "businessVerificationCheck" will be false.
+
 ```
 {
     "continueUrl" : "/test",
+    "businessVerificationCheck" : true ,
```
Comment (HarrNM) [inline README.md:59]: The changes in this ticket are only for the Sole Trader endpoint so I don't think this should be included

File: README.md [hunk 3]
```diff
@@ -122,9 +131,13 @@ an SAUTR that will be verified. By default "enableSautrCheck" will be false.
 
 All other fields must be provided.
 
+The businessVerificationCheck field allows the calling service to bypass the verification and
+continue to register where a successful match is found. By default "businessVerificationCheck" will be false.
+
 ```
 {
     "continueUrl" : "/test",
+    "businessVerificationCheck" : false ,
```
Comment (HarrNM) [inline README.md:140]: I think these have been set to true by default, to not break the current Sole Trader flows

File: README.md (hunk 2 above)
Comment (refaque-ahmad) [inline README.md:59]: Does businessVerificationCheck check box appear for individual journey?

**Review by refaque-ahmad — APPROVED** [review]
LGTM

**Pull request merged**